.github/scripts/actionlint.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+#################################################################
+#    Lint workflows with https://github.com/rhysd/actionlint    #
+#################################################################
+
+# v1.7.7
```

Comment:
Pinned this to a specific version for the same reason as all the immutable actions.